### PR TITLE
Remove unused import in rector.php, run static analysis against rector.php

### PR DESCRIPTION
**Pull request by ben-challis**

### CODE REVIEW

File: rector.php [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 declare(strict_types=1);
 
-use Rector\CodeQuality\Rector\ClassMethod\DateTimeToDateTimeInterfaceRector;
```
Comment (marmichalski) [inline rector.php:5]: can we have cs-fixer configured to run over this file?

Comment (marmichalski) [reply rector.php:5]: (and others in root directory?)

Comment (marmichalski) [reply rector.php:5]: ((and also phpstan))

Comment (ben-challis) [reply rector.php:5]: https://github.com/Lendable/composer-license-checker/pull/63/commits/9e1d6e9d5122dd6b871335b48992ce010c3e36a5

**Review by marmichalski — APPROVED** [review]

**Pull request merged**